.github/workflows/ci.yml

```diff
@@ -1,8 +1,8 @@
-name: ci 
+name: ci
 on:
   push:
     branches:
-      - master 
+      - master
```

Comment:
Nit: I'd lean towards taking this out to encourage use of `main`

Reply:
Removed main.